From f4b39613c3c345e33fa749911f34127cf91c07fc Mon Sep 17 00:00:00 2001
Message-Id: <f4b39613c3c345e33fa749911f34127cf91c07fc@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Wed, 26 Apr 2017 08:41:13 -0400
Subject: [PATCH] logical: Increase the size of the data to wipe

https://bugzilla.redhat.com/show_bug.cgi?id=1439132
https://bugzilla.redhat.com/show_bug.cgi?id=1373711

Since a sector size may be larger than 512 bytes, let's just increase
the size to wipe to 1MB rather than 2KB

(cherry picked from commit d942bf6e9e8e3991808ca5185098257e84acab5d)
Signed-off-by: John Ferlan <jferlan@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/storage/storage_backend_logical.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/storage/storage_backend_logical.c b/src/storage/storage_backend_logical.c
index a8650368b..ed26c2450 100644
--- a/src/storage/storage_backend_logical.c
+++ b/src/storage/storage_backend_logical.c
@@ -100,7 +100,7 @@ virStorageBackendLogicalInitializeDevice(const char *path)
      * a whole disk as a PV. So we just blank them out regardless
      * rather than trying to figure out if we're a disk or partition
      */
-    if (virStorageBackendZeroPartitionTable(path, 4 * PV_BLANK_SECTOR_SIZE) < 0)
+    if (virStorageBackendZeroPartitionTable(path, 1024 * 1024) < 0)
         return -1;
 
     /*
-- 
2.12.2

